client/my-sites/comments/comment-list/index.jsx

```diff
@@ -31,7 +37,56 @@ export class CommentList extends Component {
 		}
 	}
 
+	getNoticeContent = ( commentId, newStatus, previousStatus ) => {
+		this.props.removeNotice( `${ commentId }-newStatus` );
+
+		const { translate } = this.props;
+		const options = {
+			button: translate( 'Undo' ),
+			duration: 5000,
+			id: `${ commentId }-newStatus`,
+			isPersistent: true,
+			onClick: () => this.setCommentStatus( commentId, previousStatus ),
+		};
+
+		switch ( newStatus ) {
+			case 'approved':
+				return {
+					type: 'is-success',
+					message: translate( 'Comment approved.' ),
+					options,
+				};
+			case 'unapproved':
+				return {
+					type: 'is-info',
+					message: translate( 'Comment unapproved.' ),
+					options,
+				};
+			case 'spam':
+				return {
+					type: 'is-warning',
+					message: translate( 'Comment marked as spam.' ),
+					options,
+				};
+			case 'trash':
+				return {
+					type: 'is-error',
+					message: translate( 'Comment moved to trash.' ),
+					options,
+				};
+			case 'delete':
+				return {
+					type: 'is-error',
+					message: translate( 'Comment deleted permanently.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **86** times:
`translate( 'Permanently Delete Comment' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/comments/actions-transitions-notices)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).